app/views/user_mailer/reset_link.text.erb

```diff
@@ -0,0 +1,10 @@
+You're receiving this e-mail because you or someone else has requested a password reset for your SSID account.
+
+Click the link below to reset your password:
+<%= @reset_link %>
+
+If you did not request a password reset you can safely ignore this email.
+
+Thank you.
+
+The SSID Team
```

Comment:
Use constants instead of strings? "The SSID Team" => SSID_Maintainer

Reply:
Same as above.